cmd/cti/cti.go

```diff
@@ -281,7 +278,7 @@ func mainFn() int {
 			slog.Error(`                |                   `)
 		}
 		if errors.As(err, &cmdErr) && cmdErr.Inner != nil {
-			slog.Error("Command failed", slogex.ErrorWithTrace(cmdErr.Inner))
+			slog.Error("Command failed", stacktrace.ErrToSlogAttr(cmdErr.Inner))
```

Comment:
Please add an option to ensure duplicates in traces: https://github.com/acronis/go-stacktrace/blob/main/traces.go#L27